File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("world-wonders_sdk")

describe("WorldWondersSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# WorldWonders SDK exists test

require "minitest/autorun"
require_relative "../WorldWonders_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = WorldWondersSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
